pkgs/development/tools/analysis/hopper/default.nix

```diff
@@ -1,6 +1,21 @@
-{ stdenv, fetchurl, pkgs, lib }:
+{ mkDerivation, stdenv, fetchurl, pkgs, lib, qtbase }:
```

Comment:
Do not pass in `pkgs`, instead have a parameter for each required dependency

Reply:
done